documentation/learnmore/existing.md

```diff
@@ -51,6 +116,47 @@ against the empty databases (by altering the [url](/documentation/configuration/
 
 As soon as you add a new migration, it can be applied identically to any of your databases.
 
+### Example
+
+Looking at our instances, we have Production baselined and in line with our migration scripts. The Development instance has been cleaned and so is not in line. If we now run migrate against each instance it will align them. Migrate isn't strictly needed at this point for Production, however it will highlight that there is no work to be done as the baseline has been configured correctly.
+
+First we should migrate our Development instance.
+
+<pre class="console">&gt; flyway migrate -url="jdbc:postgresql://instances.spawn.cc:&lt;DevPort&gt;/pagila" \
+  -user="&lt;DevUsername&gt;" -password="&lt;DevPassword&gt;" -locations="filesystem:."</pre>
+
+<pre class="console">Database: jdbc:postgresql://instances.spawn.cc:&lt;Port&gt;/pagila (PostgreSQL 11.0)
+Successfully validated 1 migration (execution time 00:00.087s)
+Creating Schema History table "public"."flyway_schema_history" ...
+Current version of schema "public": << Empty Schema >>
+Migrating schema "public" to version "1 - baseline migration"
+
+16 rows affected
+600 rows affected
+109 rows affected
+6 rows affected
+
```

Comment:
Remove `n rows affected` as in previous comment

Reply:
Done